05/DB.test.js

```diff
@@ -1 +1,133 @@
-import DB from './DB';
\ No newline at end of file
+import DB from './DB';
+
+let DBClass;
+
+beforeEach(() => {
+	DBClass = new DB();
+});
+describe('DB insert func', () => {
+	it('func insert return correctly insert data', async () => {
+		const objectTest = {
+			say: 'hello',
+			id: 10,
+		};
+		await DBClass.insert(objectTest);
+
+		const rows = await DBClass.getRows();
+		const containsObjectTest = rows.some(row => row.id === objectTest.id);
+
+		expect(containsObjectTest).toBe(true);
+	});
+
+	it('func insert should throw an error when id is a string', async () => {
+		const objectTest = {
+			say: 'hello',
+			id: 'id',
+		};
+
+		try {
```

Comment:
Jeśli mamy warunkowe działanie (u Ciebie to try ... catch) to warto użyć `expect.assertions(1)`, aby mieć pewność, że przynajmniej jedno porównanie zostanie wykonane. W testach brak porównania to test poprawny więc może to być mylące.